atuin/src/command/client/search/interactive.rs

```diff
@@ -690,8 +701,12 @@ pub async fn history(
     }
 
     if index < results.len() {
+        let mut command = results.swap_remove(index).command;
+        if accept {
+            command = String::from("__atuin_accept__:") + &command;
```

Comment:
For non zsh shells, this will still be output, right? I think we should use a secondary gate on each supported shell `ATUIN__SUPPORTS_ACCEPT` or something

Reply:
oh oof that's true. yeah I'll do something like that, thank you

Reply:
ended up just checking for existence of `ZSH_VERSION`